tools/export/uvision4.py

```diff
@@ -98,7 +95,7 @@ def generate(self, progen_build=False):
                 project_data['common']['macros'].pop(i)
             i += 1
         project_data['common']['macros'].append('__ASSERT_MSG')
-        project_data['common']['build_dir'] = join(project_data['common']['build_dir'], 'uvision4')
+        project_data['common']['build_dir'] = project_data['common']['build_dir'] + '\\' + 'uvision4'
```

Comment:
I think @0xc0170 stumbled upon bug in uVision that '/' is not accepted as valid path separator.